Lesson05_Task01.py

```diff
@@ -0,0 +1,71 @@
+"""
```

Comment:
(venv) [antoha@dlhn test_project]$ ./bin/lint ./T-HOMETASKS/L05/l05_h01_MaximFedoseev.py
./T-HOMETASKS/L05/l05_h01_MaximFedoseev.py:1:1: D205 1 blank line required between summary line and description
./T-HOMETASKS/L05/l05_h01_MaximFedoseev.py:1:1: D400 First line should end with a period
./T-HOMETASKS/L05/l05_h01_MaximFedoseev.py:50:59: C812 missing trailing comma
./T-HOMETASKS/L05/l05_h01_MaximFedoseev.py:59:80: E501 line too long (106 > 79 characters)
./T-HOMETASKS/L05/l05_h01_MaximFedoseev.py:70:11: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L05/l05_h01_MaximFedoseev.py:70:80: E501 line too long (104 > 79 characters)
./T-HOMETASKS/L05/l05_h01_MaximFedoseev.py:71:11: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L05/l05_h01_MaximFedoseev.py:71:64: W292 no newline at end of file